app/models/deletion.rb

```diff
@@ -30,7 +30,7 @@ def record_metadata
   end
 
   def expire_api_memcached
-    ["deps/v1/#{rubygem}", "info/#{rubygem}", "versions", "names"].each do |key|
+    ["deps/v1/#{rubygem}", "versions", "names"].each do |key|
```

Comment:
but that's before it's committed, so couldn't there be a race where fastly re-requests the page before the commit is done?